src/panels/config/automation/action/ha-automation-action.ts

```diff
@@ -213,6 +165,43 @@ export default class HaAutomationAction extends LitElement {
     }
   }
 
+  private _addActionDialog() {
+    showAddAutomationElementDialog(this, {
+      type: "action",
+      add: this._addAction,
+      clipboardItem: getType(this._clipboard!.action),
```

Comment:
Clipboard can be undefined. We should do that every clipboard is used.
```suggestion
      clipboardItem: getType(this._clipboard?.action),
```